Makefile

```diff
@@ -140,7 +140,7 @@ pkg-clean:
 rpm dpkg: pkg-clean source
 	+utils/build-$@.sh -t $(SRCVERSION) -s $(DESTDIR)/pmdk -w $(DESTDIR) -o $(CURDIR)/$@\
 			-e $(EXPERIMENTAL) -c $(BUILD_PACKAGE_CHECK) -r $(BUILD_RPMEM)\
-			-f $(TEST_CONFIG_FILE) -n $(NDCTL_ENABLE)
+			-f $(TEST_CONFIG_FILE) -n $(NDCTL_ENABLE) -d $(DISTRO)
```

Comment:
We will need to make a separate variable which is either '-d $(DISTRO)' or empty if $(DISTRO) is not set.